app/styles/modules/_settings.scss

```diff
@@ -809,3 +809,113 @@ section.modal-panel {
     }
   }
 }
+
+.security-panel {
+  .security-list {
+    padding: 0;
+
+    .security-type {
+      height: 40px;
+      list-style: none;
+      margin: 10px 0;
+      position: relative;
+
+      html[dir='ltr'] & {
+        background-position: left 2px;
+      }
+
+      html[dir='rtl'] & {
+        background-position: right 2px;
+      }
+
+      .name {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        @media screen and (min-width: 400px) {
```

Comment:
did you want to define a new boundary into existing media queries that we have already, e.g.  `include respond-to('big')`

Reply:
This was taken from emails list which had to do some funky things with long emails. Clean this up a bit.